### PR TITLE
Add share button to slideshow behind experiment

### PR DESCRIPTION
**Pull request by dcravenus**

### CODE REVIEW

File: fs-album-slideshow.html [hunk 1]
```diff
@@ -358,6 +375,12 @@
     <!-- Slideshow modal -->
     <fs-modal-dialog id="slideShow" keep-fullscreen on-fs-dialog-close="handleSlideshowClose">
       <div role="dialog" id="dialogBody" class="fs-dialog__body">
+        <fs-social-menu hidden="[[!shareSlideshowEx]]">
+          <button class="share-button" on-click="toggleSocialMenu">
+            <fs-icon icon="share-batch-small" color="var(--fs-color-white)"></fs-icon>
+            Share
```
Comment (tetondev) [inline fs-album-slideshow.html:381]: Are we okay with this not referencing a translation string for now?

Comment (dcravenus) [reply fs-album-slideshow.html:381]: oops lol

Comment (dcravenus) [reply fs-album-slideshow.html:381]: i'll fix that

File: fs-album-slideshow.html [hunk 2]
```diff
@@ -358,6 +375,12 @@
...
+          </button>
+        </fs-social-menu>
         <button class="fs-dialog__close" data-dialog-dismiss="" data-test-close="" aria-label="Close">
```
Comment (tetondev) [inline fs-album-slideshow.html:384]: I know this wasn't part of your change but do we only care about `aria-labels` for English speakers? Haha, if we have the translation key available maybe we should change that too

**Review by tetondev — APPROVED** [review]

**Pull request merged**